tilequeue/worker.py

```diff
@@ -491,6 +502,15 @@ def __call__(self):
             coord = data['coord']
 
             start = time.time()
+
+            try:
+                self.inflight_mgr.unmark_inflight(coord)
```

Comment:
Nice catch! Moving this before the queue job acknowledgement will mean we're less likely to leave a tile marked as in-flight when it's not.